docs/instructor-guide/index.md

```diff
@@ -0,0 +1,26 @@
+# Instructor Guide
+
+This section provides detailed documentation for all aspects of using PrairieLearn as an instructor.
+
+To run and test question development locally, see the [local installation guide](../installing.md).
+
+For more details about configuring the four main aspects of your course:
+
+- [Courses](../course/index.md)
+- [Course Instances](../courseInstance.md)
+- [Assessments](../assessment/index.md)
+- [Questions](../question.md)
+
+!!! tip
+
+    For a refresher of what these aspects are, see the [PrairieLearn concepts](../concepts/index.md).
+
+Some other important links are:
+
+- [Access Control](../accessControl/index.md)
```

Comment:
Can you add a colon and some brief blurbs after each of these? E.g.:

```suggestion
- [Access control](../accessControl/index.md): learn how to control when and where students can access your course content.
```

Reply:
Resolved in https://github.com/PrairieLearn/PrairieLearn/pull/11371/commits/50b8c72f43aca12c6f7f3f65dd203378ad26a914.